scrapy/pqueues.py

```diff
@@ -136,9 +147,9 @@ def _active_downloads(self, slot):
 
 
 class DownloaderAwarePriorityQueue(object):
-    """ PriorityQueue which takes Downlaoder activity in account:
-    domains (slots) with the least amount of active downloads are dequeued
-    first.
+    """
+    PriorityQueue which takes Downloader activity into account: domains (slots)
+    with the least amount of active downloads are dequeued first.
```

Comment:
Both styles seem valid as per PEP 8 and PEP 257, so I wouldn’t make such a change as part of other changes. We should probably choose a style, document it in the contributing section, and _then_ start enforcing it.